File: solutions/c/rna-transcription/1/rna_transcription.c
#include "rna_transcription.h"
#include <stdlib.h>

char *to_rna(const char *dna){
    int i=0;
    char* rna = malloc(20*sizeof(char));
    while (*(dna+i)!='\0'){
        if (*(dna+i) == 'A'){
            *(rna+i)='U';            
        }
        else if (*(dna+i) == 'G'){
            *(rna+i)='C';            
        }
        else if (*(dna+i) == 'C'){
            *(rna+i)='G';            
        }
        else if (*(dna+i) == 'T'){
            *(rna+i)='A';            
        } 
        i++;
    }
    return rna;
}

File: solutions/python/rna-transcription/1/rna_transcription.py
def to_rna(dna_strand):
    rna=""
    for character in dna_strand:
        if character == 'A':
            rna=rna+'U'
        elif character == 'T':
            rna=rna+'A'
        elif character == 'C':
            rna=rna+'G'
        elif character == 'G':
            rna=rna+'C'
    return rna
